cluster/ci/config/mirroring/tekton/tekton_v0_3

```diff
@@ -0,0 +1,8 @@
+registry.svc.ci.openshift.org/openshift/knative:tektoncd-pipeline-bash:v0.3.1 quay.io/openshift-pipelines/tektoncd-pipeline-bash:v0.3.1
```

Comment:
the CI images are currently weirder : `registry.svc.ci.openshift.org/openshift/knative:tektoncd-pipeline-bash` (no "version"). 

https://github.com/openshift/release/pull/3609 is fixing that, but also changing the prefix name, so in the future, images will be `registry.svc.ci.openshift.org/openshift/tekton-v0.3.1:tektoncd-pipeline-bash` instead

Reply:
Updated